lightning/src/ln/features.rs

```diff
@@ -187,7 +187,7 @@ impl<T: sealed::Context> Features<T> {
 	pub(crate) fn requires_unknown_bits(&self) -> bool {
 		self.flags.iter().enumerate().any(|(idx, &byte)| {
 			(match idx {
-				0 => (byte & 0b00010100),
+				0 => (byte & 0b01000100),
```

Comment:
Oh can you add comment like I was suggesting here https://github.com/rust-bitcoin/rust-lightning/pull/434#discussion_r377866024